File: 240119/007/007.py
class Mission:
    def __init__(self, code, point, time):
        self.code = code
        self.point = point
        self.time = time

c, p, t = input().split()
mission1 = Mission(c,p,t)
print("secret code :", mission1.code)
print("meeting point :", mission1.point)
print("time :", mission1.time)

File: 240424/007/007.cpp
#include <iostream>
#include <string>
#include <algorithm>
#include <functional>
using namespace std;

class Spy {
    public:
        string code;
        char location;
        int time;

        Spy(string code, char location, int time) {
            this->code = code;
            this->location = location;
            this->time = time;
        }
};

int main() {
    Spy s = Spy("aa",'a',0);
    cin >> s.code >> s.location >> s.time;
    cout << "secret code : " << s.code << endl
         << "meeting point : " << s.location << endl
         << "time : " << s.time;
    return 0;
}